code/data-tracker/data-tracker.css

```diff
@@ -137,4 +137,133 @@ a.small-button {
   visibility: hidden;
   height: 0px;
   margin-bottom: 0 !important;
+}
+
+/***************************/
+/**** Step Indicator *******/
+/***************************/
+.stepindicator{
+  width: 100%;
+  z-index: 1;
+  display: flex;
+  position:static;
+}
+.label-stepindicator{
+  display:inline-block;
+  width: 1.5em;
+  height:1.5em;
+  background: #005EA2;
+  border-radius: 50%;
+  text-align: center;
+  font-weight: bold;
+  color: white;
+  line-height: 1.5em;
+  font-size:1.5em;
+}
+
+.active-label-stepindicator{
+  font-weight: bold;
+  font-size:1.5em;
+  margin-left:20px;
+}
+.progressbar{
+  counter-reset: step;
+}
+
+/* This is for the Circle number */
+.progressbar li:before{
+  content:counter(step);
+  counter-increment: step;
+  width: 30px;
+  height: 30px;
+  border: 2px solid #bebebe;
+  display:block;
+  border-radius: 50%;
+  line-height: 26px;
+  background: white;
+  color: #bebebe;
+  text-align: center;
+  font-weight: bold;
+}
+
+/* This is for the bars */
+.progressbar li{
+  list-style-type: none;
+  float: left;
+  width: 32%;
+  position: relative;
+  text-align: left;
+  margin-left: -5px;
+}
+.progressbar li + li {
+  margin-top: unset;
+}
+
+.progressbar li:after{
+  content: '';
+  position: absolute;
+  width:100%;
+  height: 8px;
+  background: #979797;
+  top: 10px;
+  left: -90%;
+  z-index: -1;
+}
+
+/* Hides the first child bar */
+.progressbar li:first-child:after{
+  content: none;
+}
+
+.progressbar li.active + li:after{
+  background: #005EA2;
+}
+
+.progressbar li.active::before{
+  border: 2px solid white;
+  background: #005EA2;
+  color: white;
+}
+.progressbar li.active{
+  font-weight: bold;
+  color: #005EA2;
+}
+.progressbar li.inactive{
+  font-weight: bold;
+  color: #163f6e;
+}
+.progressbar li.inactive + li:after{
+  background: #163f6e;
+}
+
+.progressbar li.inactive::before{
+  border-color: white;
+  background: #163f6e;
+  color: white;
+}
+
+/* For mobile screens step indicator */ 
+@media only screen and (max-width: 600px) {
+  .progressbar li:before{
+    margin-top: -33px;
+    line-height: 18px;
+    width:20px;
+    height:20px;
+    border-width: 1px;
+  }
+  .progressbar li.active::before{
+    border-width: 1px;
+  }
+  
+  .progressbar li, .progressbar li.active, .progressbar li.inactive {
+    font-size: 11px;
+    padding-top: 15px;
+  }
+  
+  .progressbar li:after{
+    border-left: 1px solid white;
+    height: 10px;
+    margin-top: -2em;
+  }
+
```

Comment:
The closing bracket on line 269 is on its own, I would either get rid of the blank line 268 or add a comment on line 269 thats indicating it closes the `@media` mobile styling

Reply:
Made that change here 9359d02a9daf611ba00611e0f86daa5906e68552! Thanks Chrispin.